code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/style/ImageOnlyUIStyle.kt

```diff
@@ -0,0 +1,81 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepcomposeui.style
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import com.adobe.marketing.mobile.aepcomposeui.AepUIConstants
+
+/**
+ * Class representing the style for a small image AEP UI.
+ *
+ * @param cardStyle The style for the card.
+ * @param rootRowStyle The style for the root row.
```

Comment:
these docs for these unused params (rootRowStyle, textColumnStyle, titleTextStyle, bodyTextStyle, buttonRowStyle, and buttonStyle) should be removed